features/works/work_edit_multiple.feature

```diff
@@ -28,22 +28,54 @@ Feature: Edit Multiple Works
       And I should see "Lovely"
 
   Scenario: I can edit multiple works at once
-    Given I am logged in as "author"
-      And I post the work "Glorious" with fandom "SGA"
-      And I post the work "Excellent" with fandom "Star Trek"
-      And I go to my works page
+  Given I am logged in as "author"
```

Comment:
Should we stick to the original indenting here, since the other tests in this file appear to use it?

Reply:
Fixed